pages/about.html

```diff
@@ -0,0 +1,105 @@
+
+  <title>About ATR</title>
+</head>
+<body>
+  <header class="header">
+    <div class="brand-logo">Brand</div>
+    <button type="button" aria-label="Mobile navigation toggle button" class="nav-toggle">
+      <span aria-hidden="true" class="hamburger"></span>
+    </button>
+    <nav class="nav">
+      <ul class="nav-list">
+        <li class="nav-list-item"><a href="../index.html" class="nav-list-link link">Home</a></li>
+        <li class="nav-list-item"><a href="../index.html#program-section" class="nav-list-link link">Program</a></li>
+        <li class="nav-list-item"><a href="../index.html#join" class="nav-list-link link">Join</a></li>
+        <li class="nav-list-item"><a href="#partner-section" class="nav-list-link link">Partners</a></li>
+        <li class="nav-list-item"><a href="#" class="nav-list-link link">News</a></li>
+        <li class="nav-list-item"><a href="#" class="nav-list-link link">AT Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <!-- Headline -->
+    <section class="headline-section">
+      <h1 class="headline-title">Africa Tech Report</h1>
+      <p class="headline-description">
+        Over the past 18 months, the Africa Tech Report 
+        has implemented an ambitious umbrella initiative 
+        to connect the African continent to broadband by 2030 
+        and leverage digital tools for economic transformation, 
+        creating better jobs for more people.
+      </p>
+      <p class="contact-us">
+        Please contact us per Email for any further questions about The Africa Tech Report.
+        <a href="mailto:contact@atr.org" class="contact-mail">contact@atr.org</a>
+      </p>
+    </section>
+    <!-- Logo section -->
+    <section class="logo-section">
+      <h2 class="logo-section-title">The Africa Tech Report Logo</h2>
+      <p class="logo-description">
+        The logo of The Africa Tech Report was decided through a 
+        competition that took place from June 8th to July 7th of the last year.
+      </p>
+      <div class="big-logo">ATR</div>
+    </section>
+    <!-- Editions section -->
+    <section class="editions-section">
+      <h2 class="editions-section-title">See the past Africa Tech Reports</h2>
+      <p class="editions-description">
+        Take a look at the last two Africa Tech Report Editions.
+      </p>
+      <ul class="editions-list list">
+        <li class="editions-list-item">
+          <a href="#" target="-blank" class="past-editions">
+            <span class="year">2020</span>
+            Edition of The Africa Tech Report
+          </a>
+        </li>
+        <li class="editions-list-item">
+          <a href="#" target="-blank" class="past-editions">
+            <span class="year">2021</span>
+            Edition of The Africa Tech Report
+          </a>
+        </li>
+      </ul>
+    </section>
+    <!-- Partners section -->
+    <section class="partner-section" id="partner-section">
```

Comment:
- [ ] If you take a look at the project desktop design, (but in the mobile size screen the vice versa) you can notice that the Partners section is placed on the `home `page, not on the `about `page. Let's change its placement.